docs/examples.rst

```diff
@@ -1,17 +1,182 @@
+*********
 Examples
-========
-
-.. toctree::
-   :maxdepth: 1
-
-   Launching jobs on Google Cloud <https://github.com/google/flax/tree/main/examples/cloud>
-   ImageNet Classification <https://github.com/google/flax/tree/main/examples/imagenet>
-   Language Modeling (lm1b) <https://github.com/google/flax/tree/main/examples/lm1b>
-   MNIST Classification <https://github.com/google/flax/tree/main/examples/mnist>
-   Part-of-Speech Tagging <https://github.com/google/flax/tree/main/examples/nlp_seq>
-   Predicting Biological Activities of Molecules with Graph Neural Networks <https://github.com/google/flax/tree/main/examples/ogbg_molpcba>
-   Proximal Policy Optimization <https://github.com/google/flax/tree/main/examples/ppo>
-   Seq2Seq: Addition <https://github.com/google/flax/tree/main/examples/seq2seq>
-   SST-2 Classification <https://github.com/google/flax/tree/main/examples/sst2>
-   Basic VAE <https://github.com/google/flax/tree/main/examples/vae>
-   Machine Translation <https://github.com/google/flax/tree/main/examples/wmt>
\ No newline at end of file
+*********
+
+Core examples
+##############
+
+
+Core examples are hosted on the Flax repo in the `examples <https://github.com/google/flax/tree/main/examples>`__ directory.
+
+Each example is designed to be **self-contained and easily forkable**, while
+reproducing relevant results in different areas of machine learning.
+
+As discussed in `#231 <https://github.com/google/flax/issues/231>`__, we decided to go for a standard pattern for all examples
+including the simplest ones (like MNIST). This makes every example a bit more
+verbose, but once you know one example, you know the structure of all of them.
+Having unit tests and integration tests is also very useful when you fork these
+examples.
+
+Some of the examples below have a link "🕹Interactive🕹" that lets you run them
+directly in Colab.
+
+Image classification
+********************
+
+- `MNIST <https://github.com/google/flax/tree/main/examples/mnist/>`__ -
```

Comment:
For every link to an example on Github, could we add a `flax/examples/xyz/` link, and/or an emoji?

This would make it very clear that at this point the RTD will lead you to the actual code on Github. I think this is an important point to make, even for readers who are just glancing at the "Examples" page without reading everything. This point was clearer before because people were already looking at the `README` in the Github UI, but now that important information is almost invisible.

![image](https://user-images.githubusercontent.com/16050001/183020948-e91e7063-ae9a-45d6-b742-4c818397547e.png)

Reply:
We could show the github icon:

![Screenshot from 2022-08-12 17-17-39](https://user-images.githubusercontent.com/5862228/184451895-45defc7b-1ead-4da7-9344-aa1feab7c076.png)

Or just a generic code icon:

![Screenshot from 2022-08-12 17-18-34](https://user-images.githubusercontent.com/5862228/184451927-722e5ac0-b6d7-4124-904c-71f9a65245b2.png)

I like the github icon.

Reply:
I chose the Github icon.